src/core/net_db.cc

```diff
@@ -309,11 +316,11 @@ bool NetDb::Load() {
           it1 != end;
           ++it1) {
 #if BOOST_VERSION > 10500
-        const std::string& fullPath = it1->path().string();
+        const std::string& full_path = it1->path().string();
```

Comment:
As noted in the contributing guide, please try not to mix formatting work with non-formatting work unless it's unavoidable to do so (in this case, it's not).